ddl/backfilling.go

```diff
@@ -199,11 +240,26 @@ type backfillTaskContext struct {
 }
 
 type reorgBackfillTask struct {
+	bfJob         *BackfillJob
```

Comment:
I noticed some place variable named bJob, here it name bfJob. Do you think we only name all of them to bfJob is necessary to make them consist?